### PR TITLE
Missing "magic method" docstrings (flake8 D105)

### PR DESCRIPTION
**Pull request by peterjc**
Work towards closing #2116.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/FSSP/__init__.py [hunk 1]
```diff
@@ -95,14 +95,15 @@ def __init__(self, inStr):
             else:
                 self.ss = '0'
 
-    def __repr__(self):
+    def __str__(self):
```
Comment (MarkusPiotrowski) [inline Bio/FSSP/__init__.py]: I have read that if `__repr__` is defined and `__str__` not, `__str__` would use `__repr__` automatically.

I guess you have a good reason to do it the more 'complicated' way?

File: Bio/Wise/psw.py [hunk 2]
```diff
@@ -85,10 +86,11 @@ def __init__(self, unit, column, kind):
         self.column = column
         self.kind = kind
 
-    def __str__(self):
+    def __repr__(self):
+        """Represent the ColumnUnit object as a string for debugging."""
         return "ColumnUnit(unit=%s, column=%s, %s)" % (self.unit, self.column, self.kind)
 
-    __repr__ = __str__
+    __str__ = __repr__
```
Comment (MarkusPiotrowski) [inline Bio/Wise/psw.py]: I think this line can be omitted? (Given that what I commented above is true)

File: Bio/FSSP/__init__.py (hunk 1 above)
Comment (peterjc) [reply Bio/FSSP/__init__.py]: You're right, good point: https://docs.python.org/2/reference/datamodel.html

Let's define just ``__repr__`` here then (and remove the historic duplication)

**Comment by peterjc** [comment]
Thanks Markus, that is more elegant fix for the duplicated ``__str__``/``__repr__`` methods 👍

File: Bio/FSSP/__init__.py (hunk 1 above)
Comment (peterjc) [reply Bio/FSSP/__init__.py]: Nope, just didn't like the historic redundant double definition.

File: Bio/Wise/psw.py [hunk 3]
```diff
@@ -85,10 +86,9 @@ def __init__(self, unit, column, kind):
         self.column = column
         self.kind = kind
 
-    def __str__(self):
-        return "ColumnUnit(unit=%s, column=%s, %s)" % (self.unit, self.column, self.kind)
-
-    __repr__ = __str__
+    def __repr__(self):
+        """Represent the ColumnUnit object as a string for debugging."""
+        return "ColumnUnit(unit=%r, column=%r, kind=%r)" % (self.unit, self.column, self.kind)
```
Comment (peterjc) [inline Bio/Wise/psw.py]: As noted by Markus, using ``%r`` here to get quotes round the values, and make the output a valid fragment of Python. Also had to add the missing ``kind=`` bit too.

File: Bio/Wise/psw.py (hunk 2 above)
Comment (peterjc) [reply Bio/Wise/psw.py]: Yes, removed.

File: Bio/Wise/psw.py (hunk 3 above)
Comment (peterjc) [reply Bio/Wise/psw.py]: Shouldn't have touched this without the command line tool dependencies installed locally - updating the tests via TravisCI failures is tedious.

**Comment by peterjc** [comment]
Windows tests failure is unrelated and affects master, see #2120

@MarkusPiotrowski Is this OK to merge now?

**Comment by MarkusPiotrowski** [comment]
Yes, OK to merge now.

**Comment by peterjc** [comment]
Thank you, your comments improved things

**Pull request merged**